add-chain/flags/flags.go

```diff
@@ -93,22 +99,14 @@ var (
 	}
 	ChainIdFlag = &cli.Uint64Flag{
 		Name:     "chain-id",
-		Usage:    "ID of chain to promote",
+		Usage:    "globally unique ID of chain",
 		Required: true,
 	}
 )
 
-var (
-	L2GenesisFlag = &cli.PathFlag{
-		Name:    "l2-genesis",
-		Value:   "genesis.json",
-		Usage:   "Path to genesis json (go-ethereum format)",
-		EnvVars: prefixEnvVars("L2_GENESIS"),
-	}
-	L2GenesisHeaderFlag = &cli.PathFlag{
-		Name:    "l2-genesis-header",
-		Value:   "genesis-header.json",
-		Usage:   "Alternative to l2-genesis flag, if genesis-state is omitted. Path to block header at genesis",
-		EnvVars: prefixEnvVars("L2_GENESIS_HEADER"),
-	}
-)
+var L2GenesisHeaderFlag = &cli.PathFlag{
+	Name:    "l2-genesis-header",
+	Value:   "genesis-header.json",
+	Usage:   "Alternative to l2-genesis flag, if genesis-state is omitted. Path to block header at genesis",
+	EnvVars: prefixEnvVars("L2_GENESIS_HEADER"),
+}
```

Comment:
I don't know if we should do anything about this now, but I think if the command is invoked with this flag most likely the chain would fail the check-genesis command?

Reply:
That portion of the code has not been tested. It is code that was copied over from the `optimism/op-chain-ops/registry-data`. Do we still want to support this alternative to to the `genesis.json` input? If so, do we have an example of what one of those inputs files would look like?

Reply:
I don't know. Let's file an issue and follow up with @protolambda about whether we could streamline this / remove that code path.